src/components/ArticlePage/ArticleContainer.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import styled from "@emotion/styled";
+import backgroundImage from "../../static/svg/backgroundImage.svg";
+import { Article } from "./Article";
+import articleImage from "../../static/svg/articleImage.png";
+import { ImageHeader } from "../Header/ImageHeader";
+
+const BackgroundImage = styled.div`
```

Comment:
you should import this as an external component. are you waiting on your landing page PR to be merged, and will reuse from there?

Reply:
When landing page will be merged I change this.